### PR TITLE
Add a Mailpile sprint to the schedule

### PR DESCRIPTION
**Pull request by BjarniRunar**
I'd like to sprint on Mailpile with you guys!

### CODE REVIEW

File: content/schedule.rst [hunk 1]
```diff
@@ -493,6 +493,7 @@ Monday 21st September 2015
 |       | * MicroPython / BBC micro:bit                                        |
 |       | * `Pylint sprint`_                                                   |
 |       | * `Pi In The Sky`_                                                   |
+|       | * `Mailpile sprint`_                                                   |
```
Comment (inglesp) [inline content/schedule.rst]: Please could you remove the extra spaces before the final pipe?

**Comment by inglesp** [comment]
Apart from the comment, looks good.

**Comment by BjarniRunar** [comment]
White-space fixed, not sure whether you want me to try and resolve the conflict which has appeared in the meantime...?

**Comment by inglesp** [comment]
Thanks @BjarniRunar -- the merge conflict should be easy to resolve.  Could you give it a go?

**Comment by BjarniRunar** [comment]
Right, okay. I'll do that.

However, I'd like to point out that you've now kept my sprint off the schedule for 24 hours because of white-space and I am now one of those people who is hacking on his laptop instead of giving the speaker polite attention.

**Comment by ghickman** [comment]
@BjarniRunar – thanks for taking the time to add your sprint and fix the merge conflict.

I'd like to re-iterate what was said in the opening plenary: this is a community conference run by volunteers, many of whom are very busy during the actual conference and doing their best to get everything done in a timely fashion. Seeing comments like this is upsetting to those who have put so much time into making the conference run as smoothly as it does, please remember this when adding more things to their already large workloads.

**Comment by BjarniRunar** [comment]
Please accept my apologies, I did not mean to offend. I am criticising the process, not the people. :-)

**Comment by BjarniRunar** [comment]
So actually, to rephrase my snark as a hopefully constructive suggestion:

In the interest of timeliness, it might be good for schedule updates that do not meet style guides to be accepted anyway, with a request for a follow-up PR to make things pretty. This gets information published as quickly as possible and avoids stalls caused by conflicts in "hot spots" of the schedule.

Again, thanks for all your hard work and I'm sorry I was grumpy.

**Comment by ghickman** [comment]
@BjarniRunar – thank you for your apology, it's very much appreciated =)

re style guide: the white space fix is actually a build issue where the table won't be generated correctly if the pipes don't line up. We're currently looking into a way to get Travis to fail the build for things like this so hopefully in the future you get a useful message from there when you PR.

**Comment by BjarniRunar** [comment]
Make sense, thanks for clarifying!

**Pull request merged**